phase1/master.cfg

```diff
@@ -734,8 +735,7 @@ c["builders"].append(
 )
 
 
-# NB the phase1 build factory assumes workers are single-build only
```

Comment:
I wouldn't move this comment as it is essential to bear it in mind when editing build steps in the factory.
Moving it down below makes it "invisible" in that context and it doesn't apply to the builder list preparation.